### ISSUE TITLE
Clicking on a link in Tooltip that is part of a Modal doesn't work

### ISSUE THREAD

**Issue by walkerp** [carbon-design-system/carbon#5349]
from IBM's @console/pal package.json using:
"carbon-components": "~10.9.0",
"carbon-components-react": "~7.9.0",

I have a Modal dialog with `focusTrap` set to true.
I have a Tooltip with a Link. When I click on the link, it doesn't go to that url.

Here's the Tooltip code:
```
<Tooltip
    triggerText={null}
    tabIndex={0}
    direction='bottom'
>
    <div style={{ fontSize: '.875rem' }}>
        <span>{t('resource-group-info')}</span>
        <br></br><br></br>
        <Link href='https://cloud.ibm.com/docs/resources?topic=resources-rgs' target='_blank' onClick={() => { }}>
            {t('resource-group-doc-link')}
        </Link>
    </div>
</Tooltip>
```
If I set focusTrap={false} on the Modal it works fine. But then I can't use the focusTrap on the Modal... which is needed.

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?

**Comment by walkerp** [comment]
@emyarod I recreated the problem in a simple Modal in a Code sandbox --> https://codesandbox.io/s/tooltip-bug-issue-5349-vi32l

**Comment by asudoh** [comment]
If this happens only happens with `focusTrap={false}`, I think it's a side effect of third-party [`focus-trap-react`](https://github.com/davidtheclark/focus-trap-react) library and #5260 addressed this issue given #5260 eliminates the need for such third-party library. Please stay tuned for the next release. Thanks!

**Comment by walkerp** [comment]
@asudoh Thanks for the quick response. Do you know when the next release will be available? What release number will it be that has the fix?

**Comment by asudoh** [comment]
@walkerp I don't have a good idea, but I imagine it's within the next couple of weeks. And I expect a newer release than the current one (which you can see at https://www.npmjs.com/package/carbon-components-react) will have this fix.

**Comment by tw15egan** [comment]
`10.10.0-rc`  should go out this week, and pending no problems, officially released EOW / start of next week

**Comment by briangleeson** [comment]
Just to say I also ran into this issue, but with an OverFlowMenu in the modal. The click events on the OverFlowMenuItem components werent working. Tried out `focusTrap=false` and it works fine. Good to see there's a fix almost out for this.

**Comment by abbeyhrt** [comment]
Closing this issue since it's been fixed in #5260 and is now released in v10.10!
